0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,673 @@
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Kevin Morrison"; $o isa object, has path "zewhb.java"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Kevin Morrison"; $o isa object, has path "budget_2021-08-01.xlsx"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Kevin Morrison"; $o isa object, has path "LICENSE"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Kevin Morrison"; $o isa object, has path "README.md"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Pearle Goodman"; $o isa object, has path "budget_2022-05-01.xlsx"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Pearle Goodman"; $o isa object, has path "zewhb.java"; 
+      $a isa action, has action-name "view_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Pearle Goodman"; $o isa object, has path "budget_2021-08-01.xlsx"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Pearle Goodman"; $o isa object, has path "LICENSE"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+
+match $s isa subject, has full-name "Pearle Goodman"; $o isa object, has path "README.md"; 
+      $a isa action, has action-name "modify_file"; $ac (accessed-object: $o, valid-action: $a) isa access; 
+insert $p (permitted-subject: $s, permitted-access: $ac) isa permission;
+```
+
+To execute the TypeQL statements copied from code block above:
+
+1. Ensure the [Session type] (schema / data) switch (next to the database dropdown) is set to `data`.
+2. Ensure the [Transaction type] (write / read) switch is set to `write`.
+3. Click the green “play” button.
+4. Click the “checkmark” button.
```

Comment:
Perhaps we should be specifying the instructions to execute above the data rather than below? This would need to be propagated to all other examples.

Reply:
I don't see a problem here. We are now moved on further and the code block we copied from is above now. The only below I'm seeing here is much higher, before the code-block, so it is correct to say below there. 

Whether the instructions below or above the code block is irrelevant as long as they are correct. 

If you mean consistency problem - we did the very same with schema, but it was divided in two sections for other reasons.